src/packages/client/src/generation/generateClient.ts

```diff
@@ -246,9 +246,22 @@ export async function generateClient({
     )
   }
   if (transpile) {
-    for (const filePath of Object.values(enginePath)) {
+    if (process.env.NETLIFY) {
+    	await makeDir('/tmp/prisma-engines')
+    }
+
+    for (const [binaryTarget, filePath] of Object.entries(enginePath)) {
+      console.warn(
+        `.env: ${process.env.NETLIFY}`,
+        `binary: ${binaryTarget}`,
+        `path: ${filePath}`,
+      )
+
       const fileName = path.basename(filePath)
```

Comment:
```suggestion
      const fileName = path.basename(filePath)
```

I think we can now remove this @millsp ?

Reply:
Yep, reverse my last 3 commits